modin/pandas/test/dataframe/test_reduction.py

```diff
@@ -361,3 +362,30 @@ def test_reduction_specific(fn, numeric_only, axis):
         *create_test_dfs(test_data_diff_dtype),
         lambda df: getattr(df, fn)(numeric_only=numeric_only, axis=axis),
     )
+
+
+@pytest.mark.parametrize("sort", [True, False])
```

Comment:
There is `bool_arg_values` which also includes `None` in the list of values to test that it doesn't cause problems.
What about other value_counts parameters, are they supported?

Reply:
added all of the `value_counts` parameters to the test